src/app/pages/turnos/calendario/turnos-calendario.html

```diff
@@ -15,6 +15,7 @@
                     <ion-label>
                         <h2 class="andes-list-title">{{efector.organizacion}}</h2>
                         <hr>
+                        <h3><b>Prestación: </b>{{a.tipoPrestaciones[0].term }}</h3>
                         <h3><b>Profesional:</b>
```

Comment:
Mini sugerencia para mejorar legibilidad de la prestación, lo dejo a tu criterio @kroce 
```suggestion
                        <h3 class="ion-text-uppercase"><b>Prestación: </b>{{a.tipoPrestaciones[0].term }}</h3>
                        <hr>
                        <h3><b>Profesional:</b>
```